src/clients/dotnet/TigerBeetle/Client.cs

```diff
@@ -132,4 +131,4 @@ private void Dispose(bool disposing)
             nativeClient.Dispose();
         }
     }
-}
+}
```

Comment:
needs trailing newline

Reply:
It was a change to trigger failures, not intended to be committed.